### PR TITLE
Move Audience & Goal to Positioning

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Rename "Audience & Goal" to "Positioning" (ui only)
Move it from "main" & "sales pitch" to "additional information"

### CODE REVIEW

File: apps/financiers/financiers/src/app/dashboard/title/view/view.module.ts [hunk 1]
```diff
@@ -35,24 +35,29 @@ const routes = [{
...
-      path: 'production',
-      loadChildren: () => import('@blockframes/movie/dashboard/production/production.module').then(m => m.MovieViewProductionModule),
+      path: 'artistic',
+      loadChildren: () => import('@blockframes/movie/dashboard/artistic/artistic.module').then(m => m.MovieViewArtisticModule),
       data: { animation: 2 }
     },
+    {
+      path: 'additional',
+      loadChildren: () => import('@blockframes/movie/dashboard/additional/additional.module').then(m => m.MovieViewAdditionalModule),
+      data: { animation: 3 }
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/dashboard/title/view/view.module.ts:50]: can you please check if you go from technical information to available materials, the animation plays the wrong direction. as you would go back but you are not.

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 2]
```diff
@@ -123,3 +123,35 @@ <h6>
...
+      <mat-icon svgIcon="pencil" color="primary"></mat-icon>
+    </a>
+  </h6>
+  <ul>
+    <li>
+      <strong>Target Audience </strong>
+      <missing-control [control]="goals.get('targets')" [link]="link">
+        <ng-template let-targets>
+          <ng-container *ngFor="let target of targets; let last = last">
+            <span>{{ target | titlecase }}{{ last ? '' : ', ' }}</span>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:142]: can you find a different solution and remove last? should be in the static model?

**Review by xmano — APPROVED** [review]
LGTM for PoV of E2E. Need to add steps for new fields. This will be addressed with a later test update.

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 3]
```diff
@@ -123,3 +123,35 @@ <h6>
...
+          </ng-container>
+        </ng-template>
+      </missing-control>
+      <br />
+      <strong>Social Responsibility Goals </strong>
+      <missing-control [control]="goals.get('goals')" [link]="link">
+        <ng-template let-goals>
+          <ng-container *ngFor="let goal of goals; let last = last">
+            <span>{{ goal | toLabel: 'socialGoals' }}{{ last ? '' : ', ' }}</span>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:151]: guess `| toLabel`  will handle that for you

File: libs/movie/src/lib/movie/marketplace/additional/additional.component.html [hunk 4]
```diff
@@ -101,5 +101,28 @@ <h5>Versions</h5>
...
+
+  
+  <!-- Audience & Goal -->
+  <section id="audience-goals" *ngIf="movie | hasKeys: 'audience'">
+    <h2>Positioning</h2>
+    <article *ngIf="movie.audience.targets.length">
+      <h5>Target audience</h5>
+      <mat-chip-list fxLayout="row" fxLayoutGap="8px">
+        <ng-container *ngFor="let target of movie.audience.targets">
+          <mat-chip>{{ target }}</mat-chip>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/marketplace/additional/additional.component.html:115]: no need for toLabel pipe or titlecase pipe?

File: libs/movie/src/lib/movie/marketplace/additional/route.ts [hunk 5]
```diff
@@ -6,6 +6,6 @@ export const additionalRoute: RouteDescription = {
   requireKeys: [
     'estimatedBudget', 'originalRelease', 'boxOffice', 'rating',
     'format', 'formatQuality', 'color', 'soundFormat',
-    'certifications'
+    'certifications', 'audience'
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/marketplace/additional/route.ts:9]: wrap it

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: apps/financiers/financiers/src/app/dashboard/title/view/view.module.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/dashboard/title/view/view.module.ts:50]: I'm not sure to understand. none of them are pages of dashboard.

Comment (fritzschoff) [reply apps/financiers/financiers/src/app/dashboard/title/view/view.module.ts:50]: ah sorry, thought that was on movie tunnel module. then forget it

File: libs/movie/src/lib/movie/marketplace/additional/route.ts (hunk 5 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/marketplace/additional/route.ts:9]: wrap what into what ?

**Review by fritzschoff — APPROVED** [review]

**Pull request merged**